### ISSUE TITLE
Update explanations

### ISSUE THREAD

**Issue by MadaZZ** [MadaZZ/JS-Concepts#24]
### Many existing JS code files need explanations.
Add those explanations to each file and raise a PR.
#### PR Branch: Each folder has its own branch. Check [contributing.md](https://github.com/MadaZZ/JS-Concepts/blob/master/CONTRIBUTING.md#branches) for more details.

**Comment by MadaZZ** [comment]
Hi @saurabhkanswal,
Check out the code files and see if it matches the format given in [Sample.JS](https://github.com/MadaZZ/JS-Concepts/blob/master/SampleJS.md). If not, change the files to follow the format.
I am sure you will find out a lot of such code files in the initial topics.

**Comment by nikikalwar** [comment]
Hi @MadaZZ , the codes dont have much comments , Can I open a new pull request and work on it?

**Comment by MadaZZ** [comment]
@nikikalwar, yeah sure! Just take care of the format in [SampleJS.md](https://github.com/MadaZZ/JS-Concepts/blob/master/SampleJS.md).
